tests/shell/028_init_masterfiles_version_3.18.2.sh

```diff
@@ -0,0 +1,12 @@
+set -e
+set -x
+cd tests/
+mkdir -p ./tmp/
+cd ./tmp/
+touch cfbs.json && rm cfbs.json
+rm -rf .git
+
+cfbs --non-interactive init --masterfiles=3.18.2
+grep '"name": "masterfiles"' cfbs.json
+grep '"version": "3.18.2"' cfbs.json
```

Comment:
Since 3.18.2 isn't gonna move, I think it would be nice to grep for the commit SHA as well here.

Reply:
https://github.com/cfengine/cfbs/pull/138